### PR TITLE
Create Ads block

### PR DESCRIPTION
**Pull request by EasileleM**
Create block for ads and block for item with sale.
image

### CODE REVIEW

File: client/src/components/AdsBlock/main.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    font-weight: 400;
+    color: #34404b;  
+
+    &_labeled {
+      color: #ff5912;
+    }
+  }
+
+  &__resolution {
+    opacity: 0.4;
```
Comment (miffn3) [inline client/src/components/AdsBlock/main.scss]: remember about code styling

File: client/src/App.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import './App.scss';
+import {Section} from './components/Section/index';
```
Comment (miffn3) [inline client/src/App.js]: section component?

File: client/src/App.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 
 function App() {
   return (
     <div className="App">
+      <Section />
```
Comment (miffn3) [inline client/src/App.js]: component used in main app.js?

File: client/src/components/SaleItem/index.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './main.scss';
+
+import square1 from './assets/square-1.svg';
+import square2 from './assets/square-2.svg';
+
+export class SaleItem extends React.Component {
```
Comment (miffn3) [inline client/src/components/SaleItem/index.js]: class without constructor

File: client/src/components/Section/index.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './main.scss';
+import { AdsBlock } from '../AdsBlock/index';
+import { SaleItem } from '../SaleItem/index';
+
+export class Section extends React.Component {
```
Comment (miffn3) [inline client/src/components/Section/index.js]: same as sales

File: client/src/components/Section/index.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  render() {
+    return (
+      <div className="section">
```
Comment (KirilushkinAleksandr) [inline client/src/components/Section/index.js]: Class name section is too abstract, I think you can change it to, f. e. , stock and change this div class='section' to html5 section tag

File: client/src/components/Section/main.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 1270px;
+    padding: 0 50px;
+
+    @media screen and (max-width: 1104px){
```
Comment (KirilushkinAleksandr) [inline client/src/components/Section/main.scss]: Media mixins we've made fits perfectly here

File: client/src/components/Section/main.scss [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+  justify-content: center;
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    width: 100%;
+    max-width: 1270px;
```
Comment (KirilushkinAleksandr) [inline client/src/components/Section/main.scss]: I thought we decided to put this max-width and padding out from our blocks

**Pull request merged**